internal/services/device_definition_service_mock_test.go

```diff
@@ -0,0 +1,248 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Now we have two of these files. I'm not sure this is an improvement. Can we do this the old way for now?